app/models/GndOntology.java

```diff
@@ -225,7 +231,8 @@ private static void loadTypes(String f) throws SAXException, IOException {
 				$(property).find(selector("range")).forEach(domain -> {
 					String type = domain.getAttribute("rdf:resource");
 					String shortType;
-					if (type != null && type.contains("#") && !(shortType = type.split("#")[1]).equals("Literal")) {
+					if (type != null && type.startsWith("http://d-nb.info/standards/elementset/gnd#")
+							&& !(shortType = type.split("#")[1]).equals("Literal")) {
```

Comment:
It may be nicer to reuse the constant in https://github.com/hbz/lobid-gnd/blob/master/app/models/AuthorityResource.java#L33 as prefix for "elementset/gnd#" or even put that whole String as constant into that class.